bridges/snowbridge/pallets/ethereum-client/src/impls.rs

```diff
@@ -53,18 +36,11 @@ impl<T: Config> Verifier for Pallet<T> {
 		if !receipt.contains_log(&event_log) {
 			log::error!(
 				target: "ethereum-client",
-				"💫 Event log not found in receipt for transaction at index {} in block {}",
-				proof.tx_index, proof.block_hash,
+				"💫 Event log not found in receipt for transaction",
```

Comment:
Why did you remove log params? I think this log became pretty non-relevant and useless, because you also remove input log with block_hash `Verifying message with block hash`.
How will you investigate `Event log not found in receipt for transaction` without knowing block_hash or tx?

Reply:
The reasoning behind this is on the original PR, over here: https://github.com/Snowfork/polkadot-sdk/pull/125#discussion_r1529183021 

A side-note, we don't generally have access to the parachain logs either, so we try and solve issues without having to look at the logs.

Reply:
ok, got it, and what about adding there some "identifier" from `proof.execution_proof` or `proof.execution_proof.execution_header`, does it make sense?

Reply:
I don't know if it would be useful. We have the same info in the relayer and that is more accessible to us.